### DISCUSSION TITLE
一种基于人寿保险理赔案件的团伙欺诈检测方法 | /project/claim-fraud-detection/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#637] [Announcements]
本文提出的这种方法在计算逻辑上非常简单，但是在业务逻辑上稍微有些复杂。总地来说，这是一种人人都能想得到的方法，但键者在走了许多弯路以后，选择了它并将其耐心打磨成型。

https://yuanfan.rbind.io/project/claim-fraud-detection/

**Comment by HushWay** [comment]
这个报告写的好专业，计算方法让我想到犯罪题材的电视剧，总是出现手段高超的连环杀人魔，而神探需要依靠细枝末节的共性特征来推断案件是否同一人所为，并进而锁定凶手的特征，这个模型有点像专门干这个的，还是挺酷的。

有一本日本小说《绝叫》就是围绕骗保事件展开的，主角阳子的一生特别悲惨，经历了从小缺爱、父亲负债潜逃、入职保险业被PUA，需要通过陪睡与自买业绩来提升业绩，后来沦落风尘场所，又遭遇了家暴与强暴事件、差点被杀害。在人生绝望的时刻，选择与恶人合作，利用保险业的漏洞，通过一系列车祸事故进行连环杀人来反击与骗保。为了恢复自由，又杀了恶人，甚至杀了她的母亲。前半部分太压抑了，后面的反击虽然在犯罪，却让我暗暗叫好，很爽快的感觉。这部小说也点出了很多日本的社会问题，包括性别歧视、消费主义陷阱诱人堕落、职场黑暗、人和人之间冷淡而出现孤独死（独自一人生活，去世都无人发现）以及保险业种种乱象等等。

**Reply by earfanfan** [reply]
>还是挺酷的

嗷，从我的角度看一点都不酷，因为之前走了太多难熬的弯路，没想到最后做出来的竟然是这样子的。本社畜，啊，不对，本牛马又在心里嚎叫了，一时间无数牛马梗涌上心头，又被自己逗乐了，哈哈，等你将来进入社会工作了，我可以给你讲很多牛马梗。

你提到的这本小说，女主角也太惨了吧，让我联想到了《白夜行》和《被嫌弃的松子的一生》。《白》我看的是小说版，女主角小时候很惨，但是有男主角一直在帮她，最后她失去了爱人但是得到了其他东西。《松》我是在武汉的琴台剧院看的话剧版，记得女主角是张静初演的，整个4小时的话剧里她一直被各种打，最后演完嗓子都快哑了。《白》与《松》对我来说都太有名了，都是看的时候就已经知道结局，知道最后女主角都是躯壳活着但灵魂已死。这些日本社会问题，我们国家也有，只不过因为种种原因，艺术作品探讨得相对少且不够深。

写到这里我忽然想给你安利一本书，《一个无政府主义者的意外死亡》，你有兴趣的话可以找来看看，我觉得真得写得很精彩，不过如果看不下去也没关系的说。

**Reply by HushWay** [reply]
> 哈哈，等你将来进入社会工作了，我可以给你讲很多牛马梗
哈哈，好呀好呀

《白》我很早看过，不过剧情全忘了。《松》很有名，但我没看过，唉，这个名字就挺丧的。我以前看了一点《霸王别姬》电影，没看下去，也是感觉太压抑了。

谢谢推荐，我很有兴趣去翻翻《一个无政府主义者的意外死亡》，不过如果不是你推荐，根据名字我可能会错过它，因为政府、政治这些东西总是让我有距离感

**Reply by HushWay** [reply]
确认一下，《意外死亡》你看的是话剧版本吗？

**Reply by earfanfan** [reply]
不是，我平时用爪机上的微信读书看书，就在那上面搜名字，找到一个版本就看了。

《霸王别姬》我是上大学的时候看的，那时候还是隔壁宿舍的同学先看了，然后她们一起讨论的时候我也好奇，于是我们宿舍的也一起看了，然后大家再一起讨论，因为那时候年轻加上朋友一起看，所以压抑氛围被抵消了不少。

**Reply by HushWay** [reply]
我找了一下，都是话剧版本的，你看的不是这种吗？它长这样
![截图](https://raw.githubusercontent.com/HushWay/Typora-img/main/img/20240920142847.png)

**Reply by earfanfan** [reply]
我看的就是这种的。哈，我明白了，我的锅，我之前误解你说的话剧版本是指真人表演的话剧。这本书本身就是一个话剧的剧本。

**Reply by HushWay** [reply]
Okk
